libs/partners/google-genai/langchain_google_genai/llms.py

```diff
@@ -76,8 +80,12 @@ def _completion_with_retry(
         )
         try:
             if is_gemini:
+                print(f"kwargs : {kwargs}")
                 return llm.client.generate_content(
-                    contents=prompt, stream=stream, generation_config=generation_config
+                    contents=prompt,
+                    stream=stream,
+                    generation_config=generation_config,
+                    safety_settings=kwargs.pop("safety_settings", None),
```

Comment:
how are safety_settings passed to the kwargs?

Reply:
Added logic for safety setting to _generate()

Reply:
and `_stream`?

Reply:
Added